src/renderers/testing/stack/ReactTestRendererStack.js

```diff
@@ -27,11 +27,12 @@ var invariant = require('invariant');
 
 import type { ReactElement } from 'ReactElementType';
 import type { ReactInstance } from 'ReactInstanceType';
+import type { ReactText } from 'ReactTypes';
 
 type ReactTestRendererJSON = {
   type: string,
-  props: { [propName: string]: string },
-  children: null | Array<string | ReactTestRendererJSON>,
+  props: { [propName: string]: any },
+  children: null | Array<ReactText | ReactTestRendererJSON>,
```

Comment:
since stack and children behave in slightly different ways I do think that numbers are still numbers at this point, but once again I’ll need to confirm in a short while.